### PR TITLE
Feature/documentation

### PR DESCRIPTION
**Pull request by alamarre3**
## Description
This pull request match with [proc_control #33](https://github.com/sonia-auv/proc_control/pull/33)
- Now EKF support parametric model for states estimation. Its give better results for AUV7. 
- Added dvl_lost_override flag in configs files. When set to false, It should increase position estimation when DVL is lost,
- Correct drag equation to be strictly positive,
- tweak AUV7 Rg to improve roll/pitch stability.
- update readme and added more doc in the code

## Fixes
- Remove docker experimental feature in github action.

## How has this been tested ?
Tested on AUV7 but not on AUV8. Those modification should not decrease the performance of AUV8, but I suggest to test it before merging it to master. For RoboSub, you can set AUV8 on master and AUV7 on devlop

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -2,5 +2,9 @@
     "python.autoComplete.extraPaths": [
         "/home/alexemdesgagne/Workspaces/ros_sonia_ws/devel/lib/python2.7/dist-packages",
         "/opt/ros/melodic/lib/python2.7/dist-packages"
+    ],
+    "python.analysis.extraPaths": [
+        "/home/alexemdesgagne/Workspaces/ros_sonia_ws/devel/lib/python2.7/dist-packages",
```
Comment (CaroBoud) [inline .vscode/settings.json]: I see that you wrote the path to your personnal directory, is it something mandatory ?

**Review by CaroBoud — APPROVED** [review]

**Review by CaroBoud — CHANGES_REQUESTED** [review]
This pr, is it for the auv7 or auv8 ? I thought it was for AUV7, but I see changes for auv8.

**Comment by alamarre3** [comment]
@CaroBoud this PR is not for an AUV in particular. I have update config files for both sub to match the ones in proc_control to avoid confusion. But it is important mentioning that the version in this repo is for debug only

**Review by CaroBoud — APPROVED** [review]
Seeams good to me !

**Pull request merged**